aztk/node_scripts/scheduling/job_submission.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import time
+
+import azure.batch.models as batch_models
+import yaml
+
+from aztk.node_scripts.core import config
+from aztk.node_scripts.install.pick_master import get_master_node_id
+from aztk.node_scripts.scheduling import common, scheduling_target
+from aztk.spark.models import ApplicationState
+from aztk.utils import constants
+
+
+def read_downloaded_tasks():
+    tasks_path = []
+    for file in os.listdir(os.environ["AZ_BATCH_TASK_WORKING_DIR"]):
+        if file.endswith(".yaml"):
+            tasks_path.append(os.path.join(os.environ["AZ_BATCH_TASK_WORKING_DIR"], file))
+
+    tasks = []
+    for task_definition in tasks_path:
+        with open(task_definition, "r", encoding="UTF-8") as stream:
+            try:
+                tasks.append(yaml.load(stream))
+            except yaml.YAMLError as exc:
+                print(exc)
```

Comment:
print => log

Reply:
This is captured in the task output and uploaded to storage. For user visibility into task errors, I think we should upload errors.